packages/fxa-settings/src/pages/InlineRecoverySetup/container.tsx

```diff
@@ -58,11 +58,12 @@ export const InlineRecoverySetupContainer = ({
 
   const verifyTotpHandler = useCallback(async () => {
     const code = await getCode(totp!.secret);
+    const service = integration.getService();
```

Comment:
What is the difference in `serviceName` passed into props vs this? Perhaps we could remove it from props?